.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+
+#
+# Authors:
+#    - David Kauschke <david.kauschke@ingenics-digital.com>
+#
+# Copyright (c) 2024 Ingenics Digital GmbH
+#
+# SPDX-License-Identifier: MIT
+#
+#
+
+name: CI-ERA-Self-Release
+
+on:
+  workflow_dispatch:
+    inputs:
+        RELEASE_USER_MAIL:
+          description: ERA commits are created with this mail
+          default: "david.kauschke@ingenics-digital.com"
+          type: string
+          required: false
+        RELEASE_USER_NAME:
+          description: ERA commits are created with this name
+          default: "David Kauschke"
+          type: string
+          required: false
+        ERA_TEST_RUN:
+          description: Enable ERA test run
+          default: "true"
+          type: string
+          required: false
+  push:
+    branches:
+      - '*'
+      - '*/*' # Match branches with single '/'
+
+env:
+  ERA_CONFIG: era/release-config.yml
+
+jobs:
+  release-era:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Store ERA release key to deploy ERA by itself via ssh checkout
+      uses: webfactory/ssh-agent@v0.9.0
+      with:
+        ssh-private-key: |
+          ${{ secrets.ERA_RELEASE_KEY }}
+    - run: python3 -m venv .venv && . .venv/bin/activate && pip install . && pip install -r requirements-test.txt
```

Comment:
requirements.txt should be sufficient

Reply:
Unfortunately not because tox is necessary which is used inside the release-config.yml
Therefore requirements-test.txt is installed